open_feature/open_feature_client.py

```diff
@@ -199,6 +257,10 @@ def create_provider_evaluation(
             flag_evaluation_options,
         )
 
+        if not self.provider:
+            logging.info("No provider configured, using no-op provider.")
+            NoOpProvider()
```

Comment:
Should this be `self.provider = NoOpProvider()` ?

Reply:
yes, done